docs/index.md

```diff
@@ -1,17 +1,17 @@
----
-page_title: "rudderstack Provider"
-subcategory: ""
-description: |-
-  
----
-
-# rudderstack Provider
-
-TBD
-
-## Example Usage
-
-```terraform
+---

```

Comment:
Any doc outside the `guides` directory is autogenerated (by running `make docs`) so any changes will be overwritten. Terraform supports templates (stored in the `templates` directory), which are being used if additional content has to be included. The way it works is that if a `.tmpl` file exists int the `templates` directory, this template is going to be used to generate the corresponding `docs` file. For this specific file, this is the `templates/index.md.tmpl` file, so any changes should first go there, and then you need to run `make docs` to generate the final docs file.

That being said, schema related documentation is not included in the template. Instead, it is being autogenerated from the `Description` fields in the related schema go struct. This means that it is not going to be possible to format the schema as a table, like you rewrote in this case, unless we do it manually for every resource, which will be hard to maintain. I will research if there is an option to affect formatting, but for now let's keep the default formatting if that is ok.